pkg/command/context.go

```diff
@@ -1408,6 +1423,7 @@ func setTanzuCtxActiveResource(_ *cobra.Command, args []string) error {
 	}
 	ctx.AdditionalMetadata[config.ProjectNameKey] = projectStr
 	ctx.AdditionalMetadata[config.SpaceNameKey] = spaceStr
+	ctx.AdditionalMetadata[config.ClusterGroupNameKey] = clustergroupStr
```

Comment:
I think we should also update the TanzuContextKubeconfig at L1431 to reflect the clustergroup endpoint.